assets/packs/data_table/Skeleton.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Creates a loading skeleton for the data table
+ */
+export function createTableSkeleton() {
+  const skeletonHTML = `
+    <div class="w-full">
+      <table class="w-full border-separate border-spacing-0 mt-2 animate-pulse">
+        <thead>
+          <tr class="bg-slate-50 h-[1.5rem]">
+          ${createHeaderCell("w-12")}
+          ${createHeaderCell("w-32")}
+          ${createHeaderCell("w-48")}
+          ${createHeaderCell("w-20")}
+          </tr>
+        </thead>
```

Comment:
I think it's fine to skip the header, the difference from rows is very subtle, and since it's a placeholder it's barely noticeable anyway.